hnn_core/gui/_viz_manager.py

```diff
@@ -837,6 +837,18 @@ def data(self):
 
     def reset_fig_config_tabs(self, template_name=None):
         """Reset the figure config tabs with most recent simulation data."""
+        simulation_names = tuple(self.data['simulations'].keys())
+        for tab in self.axes_config_tabs.children:
+            controls = tab.children[1]
+            for ax_control in controls.children:
+                # Update the options for the simulation data selection dropdown
+                simulation_data_selection = ax_control.children[1]
```

Comment:
Possible to add a test that checks these fields are updated after running a simulation?

Also is it possible to add an `assert ax_control.children[1].name == 'Data to compare'`? Possibly in the same test

This bug was so hard to find because the order of the boxes changed, it'd be nice to ensure that this index is actually targeting what we want

Reply:
I added a unit test checking the mentioned dropdown description and values.